feat(ai-monitoring): Add AI Monitoring docs (#9807)
getsentry/sentry-docs: docs/product/ai-monitoring/getting-started/index.mdx

---
title: Set Up
sidebar_order: 0
description: "Learn how to set up Sentry AI Monitoring"
---

Sentry AI Monitoring is easiest to use with the Python SDK and an official integration like OpenAI.

![AI Monitoring User Interface](../img/pipelines-view.png)

To get started, make sure you've created a sentry project for your AI-enabled repository, and follow one of the guides
below to start sending AI data to Sentry.

## Official AI integrations

- [OpenAI](/platforms/python/integrations/openai/)

> - [Langchain](/platforms/python/integrations/langchain/)

Author: this branch isn't rebased on that change

<Alert level="note" title="Don't see your platform?">

We will be adding AI integrations continuously. You can also instrument AI manually with the Sentry Python SDK.

</Alert>

## Pipelines and LLMs

> Sentry expects that you have an orchestrator (like langchain) creating pipelines of one or more AI models (such as gpt-4)
> 
> In the AI Monitoring dashboard, we show you a table of the AI pipelines, and pull the token usage from your AI models.
> 
> If you are using OpenAI without langchain, you'll need to manually create "pipelines" with the `@ai_track` annotation.
> 
> If you are using langchain without OpenAI, you might have to manually record token usage with `record_token_usage()`.
> 
> Both of these manual helpers are documented below.

Reviewer: ```suggestion
The Sentry AI Monitoring feature relies on the fact that you have an orchestrator (like langchain) creating pipelines of one or more AI models (such as gpt-4). In the AI Monitoring dashboard, we show you a table of the AI pipelines and pull the token usage from your AI models.

If you're using OpenAI without langchain, you'll need to manually create pipelines with the `@ai_track` annotation. If you're using langchain without OpenAI, you might have to manually record token usage with `record_token_usage()`. Both manual helpers are documented below.
```
